iothub/device/src/IotHubBaseClient.cs

```diff
@@ -36,12 +35,7 @@ public abstract class IotHubBaseClient : IDisposable
         private object _twinPatchCallbackContext;
         private Func<TwinCollection, object, Task> _desiredPropertyUpdateCallback;
 
-        // Diagnostic information
-
-        // Count of messages sent by the device/ module. This is used for sending diagnostic information.
-        private int _currentMessageCount;
```

Comment:
Removed along with some other diagnostics code that is really old and we don't think is important (doesn't exist in Java client, etc.).